### PR TITLE
Research data: front-end

### LINKED ISSUE #4: researchdata app: front-end
- Develop the empty/basic HTML templates fully
- Style with CSS
- Add any JavaScript
- Adjust back-end code as necessary, e.g. updating views, model, based on developments in the front-end

### PR DESCRIPTION
**Pull request by mike-allaway**
Closes #4

### CODE REVIEW

**Comment by mike-allaway** [comment]
Welcome page:

Screenshot 2021-06-03 at 21 14 02

Map page:

Screenshot 2021-06-03 at 21 14 11

Main form:

Screenshot 2021-06-03 at 21 18 58

File: django/researchdata/views.py [hunk 1]
```diff
@@ -17,10 +17,17 @@ class StoryCreateSuccessTemplateView(TemplateView):
...
+    """
+    Class-based view to show the story detail template
+    """
+    template_name = 'researchdata/story-detail.html'
+    queryset = models.Story.objects.filter(admin_published=True)
+
+
 class StoryListView(ListView):
     """
     Class-based view to show the story list template
```
Comment (branfosj) [inline django/researchdata/views.py:33]: There should be a filter for `admin_published=True`. (I cannot add the comment to the relevant line below as it is not part of this PR.) I'm happy for that to be done in a future PR.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: django/researchdata/views.py (hunk 1 above)
Comment (mike-allaway) [reply django/researchdata/views.py:33]: This is an odd one, as the StoryListView actually loops through Locations (which don't have admin_published, as they're only managed by admins) and then in the template I find the stories for all locations. But when looping in the template, I've added a "if story.admin_published=True", so it'll still filter out stories that aren't approved.

**Review by edmondac — APPROVED** [review]
lgtm

**Review by branfosj — APPROVED** [review]

**Pull request merged**